test/CMakeLists.txt

```diff
@@ -24,6 +24,10 @@ if( NOT uppercase_CMAKE_BUILD_TYPE STREQUAL "DEBUG" )
   endforeach()
 endif()
 
+if (NOT BUILD_SHARED_LIBS)
+  add_definitions(-DBENCHMARK_STATIC_DEFINE)
```

Comment:
Same

Reply:
not sure about this one as there's multiple targets all of which need the definition.